educe/stac/tests.py

```diff
@@ -378,6 +378,18 @@ def test_ambiguity_cdu_and_parent(self):
         self.assertNotIn(lg.get_edge('a', 'd'), violations)
         self.assertNotIn(lg.get_edge('b', 'd'), violations)
 
+    def test_first_incoming(self):
+        """ a -> b -> a (backwards link violation)"""
+        lg, graph = mk_graphs('#Aab / Saba')
```

Comment:
Could you rename the EDUs in this example so the parallel with the second example immediately shows up?
Plus, these two tests could be merged into one test function (checking the same property on two different graphs).